web/cypress/e2e/utils/Transactions.ts

```diff
@@ -0,0 +1,42 @@
+import {TRawTest} from '../../../src/types/Test.types';
+import {transactionTestList} from '../constants/Transactions';
+
+interface ITransactionUtils {
+  testList: TRawTest[];
+  createTest(test: TRawTest): Promise<TRawTest>;
+  createTests(tests?: TRawTest[]): Promise<TRawTest[]>;
+  deleteTest(id: string): Promise<void>;
+  deleteTests(): Promise<void[]>;
+  waitForTransactionRun(): void;
+}
+
+const TransactionUtils = (): ITransactionUtils => ({
```

Comment:
So I decided to have this here and as it is so we don't have to keep reading Ids from selectors or having global variables, in this case, this just exports the whole object which keeps track of the test list array.